dandi/cli/cmd_organize.py

```diff
@@ -34,6 +36,20 @@
     default="auto",
     show_default=True,
 )
+@click.option(
+    "--rewrite",
+    type=click.Choice(["external-file"]),
```

Comment:
Why is this a Choice with one option?  Are there other rewrite methods that could be added in the future?

Reply:
I wanted to generalise this just in case we intend to add the functionality to change some other attributes of the nwb file before upload without change in any cmd options. 
But for now its only video files and having just one option is more user friendly. Check https://github.com/dandi/dandi-cli/pull/841/commits/d6f65220ef45180b18445bd5c4719bb01bc6c7ca